src/Sylius/Bundle/ApiBundle/Command/Cart/RemoveItemFromCart.php

```diff
@@ -18,20 +18,8 @@
 /** @experimental */
 class RemoveItemFromCart implements OrderTokenValueAwareInterface
 {
-    /** @var string|null */
-    public $orderTokenValue;
-
-    /**
-     * @psalm-immutable
-     *
-     * @var string
-     */
-    public $itemId;
-
-    public function __construct(?string $orderTokenValue, string $itemId)
+    public function __construct(public ?string $orderTokenValue, public string $itemId)
     {
-        $this->orderTokenValue = $orderTokenValue;
-        $this->itemId = $itemId;
     }
 
     public static function removeFromData(string $tokenValue, string $orderItemId): self
```

Comment:
A bit off-topic, wth is this naming convention

Reply:
🤷‍♂️

Reply:
We can create a new method and deprecated this one.

Reply:
We intend to develop this topic in the next task